integration-tests/src/test/java/oracle/weblogic/kubernetes/ItIstioDomainInImage.java

```diff
@@ -217,7 +217,7 @@ public void testIstioDomainHomeInImage() {
 
     Path archivePath = Paths.get(ITTESTS_DIR, "../src/integration-tests/apps/testwebapp.war");
     ExecResult result = null;
-    result = DeployUtil.deployUsingRest(K8S_NODEPORT_HOST, 
+    result = DeployUtil.deployToClusterUsingRest(K8S_NODEPORT_HOST, 
```

Comment:
We usually use static import. Static import DeployUtil.deployToClusterUsingRest and change this DeployUtil.deployToClusterUsingRest to deployToClusterUsingRest.

Reply:
Done